src/modules/sensors/vehicle_imu/VehicleIMU.cpp

```diff
@@ -219,7 +219,7 @@ void VehicleIMU::Run()
 
 			// rotate sensor clip counts into vehicle body frame
 			const Vector3f clipping{_accel_corrections.getBoardRotation() *
-				Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
+						Vector3f{(float)accel.clip_counter[0], (float)accel.clip_counter[1], (float)accel.clip_counter[2]}};
```

Comment:
I have this problem with `make format` too but if you try to change it CI will complain to change it back because it's using a different version of astyle. That's a good example for why keeping the versions in sync like propsed in https://github.com/PX4/containers/pull/267 makes sense.